gbpservice/neutron/services/servicechain/plugins/ncp/plugin.py

```diff
@@ -104,8 +104,14 @@ def update_servicechain_instance(self, context, servicechain_instance_id,
                     context, original_instance, 'destroy')
                 deployers = self._get_scheduled_drivers(
                     context, updated_instance, 'deploy')
-        self._destroy_servicechain_nodes(context, destroyers)
-        self._deploy_servicechain_nodes(context, deployers)
+        if original_instance == updated_instance:
```

Comment:
We can't have this change unless it is upstreamed. Therefore is it possible to do without this change until it is upstremed?